### DISCUSSION TITLE
Command adjust RGB value

### DISCUSSION THREAD

**Discussion by satimis** [ImageMagick/ImageMagick#5164] [Digital Image Processing]
Hi all,
What will be the command line to adjust RGB value
Thanks
Regards

**Comment by fmw42** [comment]
Adjust how?  What do you want to happen?  Are you asking about changing brightness or contrast or color or saturation.  Please be specific. 

Also always identify your Imagemagick version and platform/OS when asking questions.

**Comment by satimis** [comment]
Hi,
Command
$ convert -brightness-contrast 10x5 image.jpegwill adjust brightness and contrast of an image.  

$ convert  -rotate -90 image.jpegwill rotate an image 90 deg counter-clockwise

I wonder whether there is a command to adjust color RGB value of an image?
Thanks
    On Friday, May 20, 2022, 06:33:36 AM GMT+8, Fred Weinhaus ***@***.***> wrote:  
 
 


Adjust how? What do you want to happen? Are you asking about changing brightness or contrast or color or saturation. Please be specific.

**Reply by fmw42** [reply]
Perhaps you want

`convert image -modulate Brightness,Saturation,Hue result
`

**Reply by fmw42** [reply]
For new users, see

For novices, see

http://www.imagemagick.org/discourse-server/viewtopic.php?f=1&t=9620
http://www.imagemagick.org/script/command-line-processing.php
https://imagemagick.org/script/command-line-options.php
http://www.imagemagick.org/Usage/reference.html
http://www.imagemagick.org/Usage/
https://github.com/ImageMagick/usage-markdownhttps://imagemagick.org/script/porting.php#cli
https://legacy.imagemagick.org/discourse-server/index.php
